py3status/modules/net_rate.py

```diff
@@ -51,9 +63,15 @@ class Py3status:
     format = "{interface}: {total}"
     format_no_connection = ''
     hide_if_zero = False
+    initial_multi = 1024
```

Comment:
I don't think these should be parameters.  really we need to move this sort of stuff into common code so that all modules can use the same paths rather than having lots of modules doing this independently.  I've got a load of such common functions that I should really make into a branch.

From the user point of view I think specifying a unit eg show me in 'Gb/s' makes sense but the details of the algorithm feel out of place.  I'm all for simplifying configs where we can whilst still allowing flexibility for the user.